base/templates/admin/data_edit.html

```diff
@@ -0,0 +1,239 @@
+{% extends "_layouts/default.html" %}
+
+{% block custom_head %}
+
+<script src="https://apis.google.com/js/client.js"></script>
+
+{% endblock %}
+
+
+{% block content %}
+{% from "macros.html" import render_field %}
+
+
+{% if report.initialized == True %}
+
+<div class="row">
+  <div class='col-md-2'></div> <!-- /col-md-2 -->
+  <div class="row">
+
+    <div class='col-md-8'>
+      <h3>
+        <div style="text-align:center;">
+          <br> <strong> {{ report.kind }} </strong>
+          <br> <strong> {{ report.name }} </strong>
+          <hr>
+        </div>
+      </h3>
+        </div> <!-- /col-md-8 -->
+  </div> <!-- /row -->
+  <div class="row">
+    <div class='col-md-2'></div> <!-- /col-md-2 -->
+    <div class='col-md-8'>
+
+        <br>
+        <div class='row'>
+          <div class='col-md-4'> 
+            <strong> Dataset: </strong>
+          </div> <!-- /col-md-4 -->
```

Comment:
If you use `{# #}` these will be removed before being sent to the user. Not a big deal really just FYI.